ansible/playbooks/service/phpipam.yml

```diff
@@ -10,5 +10,38 @@
 
   roles:
 
+    - role: debops.php/env
+      tags: [ 'role::php', 'role::apt_preferences', 'role::logrotate' ]
+
+    - role: debops.apt_preferences
+      tags: [ 'role::apt_preferences', 'skip::apt_preferences' ]
+      apt_preferences__dependent_list:
+        - '{{ php__apt_preferences__dependent_list }}'
+        - '{{ nginx__apt_preferences__dependent_list }}'
+
+    - role: debops.logrotate
+      tags: [ 'role::logrotate' ]
+      logrotate__dependent_config:
+        - '{{ php__logrotate__dependent_config }}'
+
+    - role: debops.php
+      php__dependent_packages:
+        - '{{ phpipam__php__dependent_packages }}'
+      php__dependent_pools: [ '{{ phpipam__php5_pool }}' ]
+      when: ((phpipam__dependencies is defined and phpipam__dependencies) and
+        (phpipam__mode is defined and 'webui' in phpipam__mode))
+
+    - role: debops.ferm
+      tags: [ 'role::ferm', 'skip::ferm' ]
+      ferm__dependent_rules:
+        - '{{ nginx__ferm__dependent_rules }}'
+
+    - role: debops.nginx
+      nginx_servers: [ '{{ phpipam__nginx_server }}' ]
+      nginx_upstreams: [ '{{ phpipam__nginx_upstream_php5 }}' ]
```

Comment:
The `phpipam__*` variables here should also be updated to point to relevant dependent variables in the `nginx` role. And switch the dependent variables here to correct ones, `nginx__dependent_upstreams`, `nginx__dependent_servers`.

Reply:
done